tests/distributed/test_workspaces/test_remote_workspaces.py

```diff
@@ -215,7 +215,7 @@ def docker_compose(request):
 @pytest.mark.parametrize('docker_compose', [compose_yml], indirect=['docker_compose'])
 def test_upload_simple_non_standard_rootworkspace(docker_compose):
     f = (
-        Flow()
+        Flow(port_expose=exposed_port)
```

Comment:
where is this exposed port?

Reply:
It is defined up top (line 17) and then just used between each Flow and Client. Is that what you mean?